.github/dependabot.yml

```diff
@@ -3,7 +3,19 @@ updates:
   # Enable version updates for Gradle
   - package-ecosystem: "gradle"
     directory: "/"
-    open-pull-requests-limit: 10
+    open-pull-requests-limit: 5
+    groups:
+      npm-major-dependencies:
```

Comment:
nit: remove `npm` since this is a Gradle project. I was confused at first about whether this would only look for npm updates before realizing it was just the group name

Reply:
Oops! I will update.